packages/app/src/app/overmind/effects/vscode/Workbench.ts

```diff
@@ -201,15 +201,23 @@ export class Workbench {
       },
     });
 
-    this.appendMenuItem(MenuId.MenubarFileMenu, {
-      // Z to be the last item after vscode group 4
-      group: '4_zsandbox',
-      order: 1,
-      command: {
-        id: 'codesandbox.sandbox.fork',
-        title: '&&Fork Sandbox',
-      },
-    });
+    if (
+      this.controller.getState().editor?.currentSandbox?.teamId ===
+        this.controller.getState().activeTeam &&
+      this.controller.getState().activeWorkspaceAuthorization === 'READ'
+    ) {
+      // don't add the option to fork
```

Comment:
The implementation makes sense, but I would revert the check. I typically do this trick:

```js
!(mySpecificScenarioWhereItShouldNotWork)

// so

!(this.controller.getState().editor?.currentSandbox?.teamId ===
        this.controller.getState().activeTeam &&
      this.controller.getState().activeWorkspaceAuthorization === 'READ')
```

It makes it more readable I think. Allows you to say: "this is the specific exception and I want it to work in all other scenarios" 😄

Reply:
Made the changes :)